documentation/book/con-cluster-operator-watch-options.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// assembly-operators-cluster-operator.adoc
+// assembly-cluster-operator.adoc
+
+[id='con-cluster-operator-watch-options-{context}']
+
+= Watch options for a Cluster Operator deployment
+
+When the Cluster Operator is running, it starts to _watch_ Kafka resources for updates.
```

Comment:
Would this be better:

```suggestion
When the Cluster Operator is running, it starts to _watch_ for Kafka resources and their updates.
```

Reply:
Changed to 'watch for updates of Kafka resources'